Makefile

```diff
@@ -36,8 +36,10 @@ TEST_E2E_DIR := test/e2e
 E2E_DATA_DIR ?= $(REPO_ROOT)/test/e2e/data
 E2E_CONF_PATH  ?= $(E2E_DATA_DIR)/e2e_conf.yaml
 E2E_EKS_CONF_PATH ?= $(E2E_DATA_DIR)/e2e_eks_conf.yaml
+E2E_CONF_CNI_ANTREA ?= $(E2E_DATA_DIR)/e2e_conf_cni_antrea.yaml
 KUBETEST_CONF_PATH ?= $(abspath $(E2E_DATA_DIR)/kubetest/conformance.yaml)
 KUBETEST_FAST_CONF_PATH ?= $(abspath $(E2E_DATA_DIR)/kubetest/conformance-fast.yaml)
+KUBETEST_NETWORKPOLICY_CONF_PATH ?= $(abspath $(E2E_DATA_DIR)/kubetest/conformance-networkpoicy.yaml)
```

Comment:
typo ... `networkpoicy`

Reply:
line 42 in this yaml file is still `poicy`

Reply:
opps.. didn't save it